fenix/app/src/main/java/org/mozilla/fenix/debugsettings/ui/DebugOverlay.kt

```diff
@@ -30,7 +39,24 @@ import org.mozilla.fenix.theme.FirefoxTheme
 @Composable
 fun DebugOverlay() {
     val snackbarState = remember { SnackbarHostState() }
-    val scope = rememberCoroutineScope()
+    val drawerState = remember { DrawerState(initialValue = DrawerValue.Closed) }
+    var drawerEnabled by remember { mutableStateOf(false) }
+
+    LaunchedEffect(drawerEnabled) {
+        if (drawerEnabled) {
+            drawerState.open()
+        }
+    }
```

Comment:
because of the weirdness with Surface mentioned in the comment, I needed to add this launched effect in order to prevent some open animation weirdness on the first open